src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -51,7 +51,10 @@ public IEnumerable<Stream> GetManifests()
                 foreach (var workloadName in Directory.EnumerateDirectories(manifestDirectory))
                 {
                     var workloadManifest = Path.Combine(workloadName, "WorkloadManifest.json");
-                    yield return File.OpenRead(workloadManifest);
+                    if (File.Exists(workloadManifest))
```

Comment:
This will break one of my test. It make sense to throw. There has to be a WorkloadManifest.json under manifest folder right?

Reply:
I reverted this change.